Runtime/Hosting/ManagedHost.cs

```diff
@@ -60,6 +60,7 @@ public static napi_value InitializeModule(napi_env env, napi_value exports)
         catch (Exception ex)
         {
             Console.Error.WriteLine($"Failed to load CLR managed host module: {ex}");
```

Comment:
The `Console.Error.WriteLine()` can be changed to `Trace()` now that the error is thrown.

And same in NativeHost.cs.